images/cross-cloud/sle-hpc/ondemand/profiles.yaml

```diff
@@ -10,17 +9,14 @@ profiles:
   Azure:
     description: Azure configuration
     include:
-      - csp/azure/ondemand/hpc
       - csp/azure/addon/azure-kernel
       - csp/azure/addon/azure-tools
       - csp/azure/addon/waagent-rdma
   EC2:
     description: EC2 configuration
     include:
       - csp/ec2/addon/aws-tools
-      - csp/ec2/ondemand
   GCE:
     description: GCE configuration
     include:
       - csp/gce/addon/gce-tools
-      - csp/gce/ondemand/hpc
```

Comment:
We'll have to retain the `ondemand` includes. Firstly, because we probably want to put the config section that enables `guestregister.service` there, and secondly because we use those paths to slot in the billing tags from the private repo.